src/ert/gui/simulation/run_dialog.py

```diff
@@ -307,17 +306,18 @@ def on_snapshot_new_iteration(
     ) -> None:
         if not parent.isValid():
             index = self._snapshot_model.index(start, 0, parent)
+            iteration = cast(Node, index.internalPointer()).id_
```

Comment:
Should we cast it to Node or IterationNode? I think the point of having Node prefixed with underscore was that it would not be imported elsewhere

Reply:
changed to IterationNode. More specific is always better